lib/virt_autotest/utils.pm

```diff
@@ -319,9 +319,11 @@ sub check_guest_health {
     if (script_run("virsh list --all | grep \"$vm \"") == 0) {
         $vmstate = "ok" if (script_run("virsh domstate $vm | grep running") == 0);
     }
-    if (is_xen_host and script_run("xl list $vm") == 0) {
-        script_retry("xl list $vm | grep \"\\-b\\-\\-\\-\\-\"", delay => 10, retry => 1, die => 0) for (0 .. 3);
-        $vmstate = "ok" if script_run("xl list $vm | grep \"\\-b\\-\\-\\-\\-\"");
+    else {
+        if (is_xen_host and script_run("xl list $vm") == 0) {
+            script_retry("xl list $vm | grep \"\\-b\\-\\-\\-\\-\"", delay => 10, retry => 1, die => 0) for (0 .. 3);
+            $vmstate = "ok" if script_run("xl list $vm | grep \"\\-b\\-\\-\\-\\-\"");
+        }
```

Comment:
I think this block can be simplified using just ```elsif``` instead of ```if``` in a ```else```.

Reply:
Yes, Thanks.